gitlab/client.py

```diff
@@ -544,7 +546,7 @@ def http_request(
             path (str): Path or full URL to query ('/projects' or
                         'http://whatever/v4/api/projecs')
             query_data (dict): Data to send as query parameters
-            post_data (dict): Data to send in the body (will be converted to
+            post_data (dict|bytes): Data to send in the body (will be converted to
```

Comment:
Thanks for taking care even of the docstrings. Hopefully we can remove the duplicate typing after it's all done :)